ta/mk/ta_dev_kit.mk

```diff
@@ -61,6 +61,13 @@ libdeps += $(ta-dev-kit-dir)/lib/libutils.a
 libdeps += $(ta-dev-kit-dir)/lib/libmpa.a
 libdeps += $(ta-dev-kit-dir)/lib/libutee.a
 
+# Pass config variable (CFG_) from conf.mk on the command line
+cppflags$(sm) += $(strip \
+	$(foreach var, $(filter CFG_%,$(.VARIABLES)), \
+		$(if $(filter y,$($(var))), \
+			-D$(var)=1, \
+			$(if $(filter xn x,x$($(var))),,-D$(var)=$($(var))))))
```

Comment:
`D$(var)='$($(var))'` would be safer (although not bullet-proff) and avoid the dependency on https://github.com/OP-TEE/optee_test/pull/277.